### DISCUSSION TITLE
visual bell when tmux changing window on remote terminal

### DISCUSSION THREAD

**Discussion by danielb2** [tmux/tmux#3759] [General]
recently Kitty (or something) is triggering a visual bell everytime (or almost) I switch videos using tmux. This is so annoying I am considering disabling the visual bell, but I think VB has it's uses too. Any idea what could be causing this?

This isn't kitty specific, it's also happening in iTerm. I'm just hoping someone can help

**Comment by nicm** [comment]
I don't really know what this means. Are you sure it is a bell? Every time you change tmux window?
